src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ControllerModel.cs

```diff
@@ -6,14 +6,13 @@
 using System.Diagnostics;
 using System.Linq;
 using System.Reflection;
-using Microsoft.AspNetCore.Mvc.ActionConstraints;
 using Microsoft.AspNetCore.Mvc.Filters;
 using Microsoft.AspNetCore.Mvc.Routing;
 
 namespace Microsoft.AspNetCore.Mvc.ApplicationModels
 {
     [DebuggerDisplay("Name={ControllerName}, Type={ControllerType.Name}," +
-                     " Routes: {AttributeRoutes.Count}, Filters: {Filters.Count}")]
+                     " Filters: {Filters.Count}")]
```

Comment:
Let's simplify this to just the controller type name as well. These DD's aren't super readable as-is.